App.UserSupport/Models/Handling.cs

```diff
@@ -12,23 +12,24 @@ public class Handling
         //Client who write Handling
         private Client clientHandling;
 
-        public Handling(Client client, string firstmessage, int id)
+        public Handling(Client client, string firstmessage, int id,int idmess)
```

Comment:
why you are passing messageId to the constructor of Handling ?

Reply:
I need it, for initialization my messages in DB.